mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -742,13 +922,49 @@ impl<MakerCoin: MmCoin + CoinAssocTypes, TakerCoin: MmCoin + SwapOpsV2> State fo
             },
         };
 
+        let total_payment_value =
+            &(&state_machine.taker_volume + &state_machine.dex_fee.total_spend_amount()) + &state_machine.taker_premium;
```

Comment:
As we agreed we should include funding transaction spend fee in the total payment value of the taker. I guess you will do that in next iterations.

Reply:
Yeah, I will try to do it in the current iteration 🙂